samples/Simple.HttpPatch.Samples/Controllers/ValuesController.cs

```diff
@@ -23,7 +23,7 @@ public string Get()
         }
 
         [HttpPatch]
-        public IEnumerable<string> Patch([FromBody] Patch<Person> person)
+        public IEnumerable<string> Patch([ModelBinder(BinderType = typeof(PatchObjectModelBinder<Person>))] Patch<Person> person)
```

Comment:
can we simplify it to?
```
[ModelBinder(BinderType = typeof(PatchObjectModelBinder))
```

Reply:
I added IModelBinderProvider to simplify it